libcodechecker/server/client_db_access_handler.py

```diff
@@ -758,12 +787,24 @@ def getRunResultCount(self, run_ids, report_filter, cmp_data):
                     # There is no difference.
                     return 0
 
-            q = session.query(Report) \
-                .filter(Report.run_id.in_(run_ids)) \
-                .outerjoin(File, Report.file_id == File.id) \
-                .outerjoin(ReviewStatus,
-                           ReviewStatus.bug_hash == Report.bug_id) \
-                .filter(filter_expression)
+            isUnique = report_filter is not None and \
+                       report_filter.isUnique
+
+            if isUnique:
+                q = session.query(Report.bug_id,
+                                  Report.checker_id,
+                                  Report.severity) \
+                    .distinct(Report.bug_id,
```

Comment:
Shouldn't we use the same uniqueing here as before?
` .distinct(Report.bug_id, Report.checker_id, Report.checker_message, Report.severity, File.filename, ReviewStatus.status)`

Reply:
Yes, we have to. Thanks.